templates/consul.systemd.erb

```diff
@@ -1,16 +1,14 @@
 [Unit]
-Description=Consul Agent
-Wants=basic.target
-After=basic.target network.target
+Description=Consul agent
+Requires=network-online.target
+After=network-online.target
 
 [Service]
-User=<%= scope.lookupvar('consul::user') %>
-Group=<%= scope.lookupvar('consul::group') %>
-ExecStart=<%= scope.lookupvar('consul::bin_dir') %>/consul agent \
-  -config-dir <%= scope.lookupvar('consul::config_dir') %> <%= scope.lookupvar('consul::extra_options') %>
-ExecReload=/bin/kill -HUP $MAINPID
-KillMode=process
+EnvironmentFile=-/etc/sysconfig/consul
 Restart=on-failure
```

Comment:
Hi
 One more thing that we have to look after consul installation if in system "systemctl daemon-reload " command is used  then i do not get consul.service  automatically restart ,so service consul is hanging.thanks for collaboration

Regards
Karen